### DISCUSSION TITLE
Problem to connecting last fm

### DISCUSSION THREAD

**Discussion by Sacrylegio** [pear-devs/pear-desktop#1766] [Q&A] [answered]
After the last update, lastfm automatically disconnected me, and I noticed that the setting had also changed, so I went and put the API key of my last fm account and also the secret one, even though I restarted and lastfm I asked for authorization for this app, lastfm doesn't detect it as an added application and it doesn't work, how can I resolve this?

**Comment by Hadriang** [comment]
Same here, double checked my api key, and reconnected the app to last fm but no scrobbles since yesterday ¯\_(ツ)_/¯

**Comment by Kronos25** [comment]
Also not working here

**Comment by JellyBrick** [comment, marked as answer]
Will be fixed at the next release
